### ISSUE TITLE
Different behavior between math.evaluate and parser.evaluate

### ISSUE THREAD

**Issue by dvd101x** [josdejong/mathjs#2319]
When evaluating multiple expressions:
`
math.evaluate(["1+2","1+3"])
`
Result is:
`
[3, 4]
`
When doing the same on a parser

```
const parser = self.math.parser()
parser.evaluate(["1+2","1+3"])
```
Result is:
`
TypeError: t(e).compile is not a function. (In 't(e).compile()', 't(e).compile' is undefined)
`

**Comment by josdejong** [comment]
Thanks, good suggestion 👍  fixed now via ed0fe96 (not yet published)

**Comment by dvd101x** [comment]
That's awesome! and super fast! I'm soo greateful for this magnificent project.

**Comment by josdejong** [comment]
The fix is now published in v9.5.0
